frontstage/error_handlers.py

```diff
@@ -14,49 +13,38 @@
 
 @app.errorhandler(404)
 def not_found_error(error):  # pylint: disable=unused-argument
-    return redirect(url_for('error_bp.not_found_error_page',
-                            _external=True,
-                            _scheme=getenv('SCHEME', 'http')))
+    logger.error('Not found error', status=error.code)
+    return render_template('errors/404-error.html'), 404
 
 
 @app.errorhandler(ApiError)
 def api_error(error):
-    error.logger(error.message or 'Api failed to retrieve required data',
+    logger.error(error.message or 'Api failed to retrieve required data',
                  status=error.status_code,
                  url=error.url,
                  **error.kwargs)
-    return redirect(url_for('error_bp.server_error_page',
-                            _external=True,
-                            _scheme=getenv('SCHEME', 'http')))
+    return render_template('errors/500-error.html'), 500
 
 
 @app.errorhandler(ConnectionError)
 def connection_error(error):
     logger.error('Failed to connect to external service', url=error.request.url)
-    return redirect(url_for('error_bp.server_error_page',
-                            _external=True,
-                            _scheme=getenv('SCHEME', 'http')))
+    return render_template('errors/500-error.html'), 500
 
 
 @app.errorhandler(JWTValidationError)
 def jwt_validation_error(error):  # pylint: disable=unused-argument
-    return redirect(url_for('error_bp.not_logged_in_error_page',
-                            _external=True,
-                            _scheme=getenv('SCHEME', 'http')))
+    logger.error('JWT validation error')
+    return render_template('errors/not-signed-in.html'), 403
 
 
 @app.errorhandler(Exception)
 def server_error(error):  # pylint: disable=unused-argument
     logger.exception('Generic exception generated', exc_info=error)
-
-    return redirect(url_for('error_bp.server_error_page',
-                            _external=True,
-                            _scheme=getenv('SCHEME', 'http')))
+    return render_template('errors/500-error.html'), 500
 
 
 @app.errorhandler(InvalidEqPayLoad)
 def eq_error(error):
     logger.error('Failed to generate EQ URL', error=error.message)
-    return redirect(url_for('error_bp.server_error_page',
-                            _external=True,
-                            _scheme=getenv('SCHEME', 'http')))
+    return render_template('errors/500-error.html'), 500
```

Comment:
Every error needs to log the status code and url. I'd say the status code should be logged as `status_code` rather than just `status` as that's what's done in eQ (see https://github.com/ONSdigital/eq-survey-runner/blob/master/app/views/errors.py#L71) and it would be good to get some consistency across the platform.